README.md

```diff
@@ -85,6 +85,29 @@ Here is a quick list of what's kept the same or what's similar between `content`
   - [Shell prompt guidelines](https://developer.mozilla.org/en-US/docs/MDN/Guidelines/Code_guidelines/Shell)
 - **Formatting and layout:** the translated docs should, in general, be formatted and laid out in the same way as the en-US versions — however, if you find mistakes (e.g. incorrect Markdown syntax), please correct them in the translated file(s), and feel free to submit a PR at the mdn/content repo to fix those errors at the source as well!
 
+#### Unicode characters
+
+Code editors like VS Code have implemented features to highlight characters that are "often confused with another" and/or "invisible" (in VS Code, this is done with a yellow rectangle surrounding a character) — you can hover over each highlighted character to see its unicode hex and a potential replacement. The [Gremlins tracker for Visual Studio Code] extension ([find/install it here](https://marketplace.visualstudio.com/items?itemName=nhoizey.gremlins)) is an excellent to search for these characters as well — see after the table below for images of the VS Code and Gremlins tracker features in action. Here are some examples of "gremlins":
+
+| Character name | Unicode | Potential replacement |
+| -------------- | ------- | --------------------- |
+| Non-breaking space | `U+00a0` | Regular space (note that non-breaking spaces are [sometimes intentional](#key-localization-differences)) |
```

Comment:
Clarify that if they're lying around (not intentional), they should be replaced with regular spaces
```suggestion
| Non-breaking space | `U+00a0` | Regular space — if you see any that are "randomly placed" in a file, feel free to replace them as they're probably not meant to be there. (**Note:** Non-breaking spaces are [sometimes intentional](#key-localization-differences)) |
```